apps/backend/src/event/event.module.ts

```diff
@@ -0,0 +1,12 @@
+import { Module } from '@nestjs/common';
+import { PrismaModule } from 'nestjs-prisma';
+
+import { EventController } from './event.controller';
+import { EventService } from './event.service';
+
+@Module({
+  imports: [PrismaModule.forRoot({ isGlobal: true })],
```

Comment:
There's no need to import PrismaModule again. It's imported in the app.module with the isGlobal flag, so it's available throughout the project